### PR TITLE
voss-27 update styles from main to match template

### PR DESCRIPTION
**Pull request by amymanion**

### CODE REVIEW

File: site/_scss/site/objects/_components.scss [hunk 1]
```diff
@@ -0,0 +1,152 @@
+.grid.three {
```
Comment (ccbayer) [inline site/_scss/site/objects/_components.scss]: @amymanion split this into separate partials for each "component"; define vars for common hex colors.

File: site/community.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Community
+id: community
+---
+
+<div class="sub-page">
+    <h3>Do you want to help build Contour?</h3>
```
Comment (ccbayer) [inline site/community.html]: @amymanion this should be h1?

File: site/resources.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Resources
+id: resources
+---
+<div class="sub-page">
+    <h3>Here you will find external resources about Contour, such as videos, podcasts, and community articles.</h3>
```
Comment (ccbayer) [inline site/resources.html]: @amymanion this should be h1

File: site/resources.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                <p class="strong">Demos and Deep Dives</p>
+                <p>We have a YouTube playlist for Contour demos and code deep dives, <a href="https://www.youtube.com/playlist?list=PL7bmigfV0EqRTmmjwWm4SxuCZwNvze7se">check it out here</a>.</p>
+            </div>
+        </div>
+        <div class="col">
+            <div class="embed-responsive embed-responsive-16by9">
+                <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/jzla7xdqtwM"></iframe>
+              </div>
+            <div class="content">
+                <p><a href="https://www.youtube.com/embed/jzla7xdqtwM">Take Envoy Beyond a K8s Service Mesh - to Legacy Bare Metal & VMs + More</a></p>
```
Comment (ccbayer) [inline site/resources.html]: @amymanion ensure these link to the youtube URL, this is currently linking to the youtube `embed` url

**Review by ccbayer — CHANGES_REQUESTED** [review]
@amymanion see comments

File: site/resources.html (hunk 3 above)
Comment (amymanion) [reply site/resources.html]: Updated to h2 - there is another h1 on the page.

File: site/community.html (hunk 2 above)
Comment (amymanion) [reply site/community.html]: @ccbayer There already is an h1, but it should be an h2. Updated it.

File: site/resources.html (hunk 3 above)
Comment (amymanion) [reply site/resources.html]: @ccbayer There already is an h1, but this should be an h2. Updated it.

File: site/resources.html (hunk 4 above)
Comment (amymanion) [reply site/resources.html]: Fixed this.

**Pull request closed without merging**